perllib/FixMyStreet/Cobrand/Bromley.pm

```diff
@@ -968,6 +1049,61 @@ sub waste_check_last_update {
     return 1;
 }
 
+sub waste_get_next_dd_day {
+    my $self = shift;
+
+    my $dd_delay = 10; # No days to set up a DD
+
+    my $dt = DateTime->now;
+    my $wd = FixMyStreet::WorkingDays->new(public_holidays => FixMyStreet::Cobrand::UK::public_holidays());
+
+    my $next_day = $wd->add_days( $dt, $dd_delay );
+
+    return $next_day;
+}
+
+sub waste_get_pro_rata_bin_cost {
+    my ($self, $end, $start) = @_;
+
+    $start->truncate(to => 'day');
+    $end->truncate(to => 'day');
+    my $weeks = $end->delta_days($start)->in_units('weeks') - 1;
+
+    my $base = $self->feature('payment_gateway')->{pro_rata_minimum};
+    my $weekly_cost = $self->feature('payment_gateway')->{pro_rata_weekly};
+
+    my $cost = $base + ( $weeks * $weekly_cost );
+
+    return $cost;
+}
+
+sub waste_sub_due {
+    my ($self, $date) = @_;
+
+    my $now = DateTime->now;
+    my $sub_end = DateTime::Format::W3CDTF->parse_datetime($date);
+
+    my $diff = $now->delta_days($sub_end)->in_units('weeks');
+    return $diff < 6;
+}
+
+sub waste_sub_overdue {
+    my ($self, $date) = @_;
+
+    my $now = DateTime->now;
+    my $sub_end = DateTime::Format::W3CDTF->parse_datetime($date);
+
+    return $now > $sub_end;
+}
+
+sub garden_waste_cost {
+    my ($self, $bin_count) = @_;
+
+    $bin_count ||= 1;
+
+    return $self->feature('payment_gateway')->{ggw_cost} * $bin_count;
```

Comment:
There are a number of places that make this calculation and don't use this function.